### DISCUSSION TITLE
ARM Image Support?

### DISCUSSION THREAD

**Discussion by phyyou** [apache/kyuubi#4487] [General]
Hi, I would like to request a new feature that provides support for ARM-based images Apache Kyuubi.
Currently, in the [Docker Hub](https://hub.docker.com/r/apache/kyuubi/tags) only provides images for x86-based architectures.

Have you any plan of ARM architecture Image support?

**Comment by pan3793** [comment]
I think Kyuubi should support ARM, we can update the image build scripts to publish it to docker hub cc @ulysses-you

**Reply by ulysses-you** [reply]
yeah, we are open to support ARM architecture. It would take some time for community to support that. It would be great if you have time to join the community to work on this @phyyou

**Reply by phyyou** [reply]
Thanks! @ulysses-you

I would love to join the community and work on this feature. Can you let me know how I can get started and contribute for ARM Image support to Apache Kyuubi? I look forward to hearing your feedback 😊

**Reply by ulysses-you** [reply]
yes, just go head. Welcome @phyyou !

**Reply by yaooqinn** [reply]
https://docs.docker.com/engine/reference/commandline/buildx_create/#platform, https://docs.docker.com/build/ci/github-actions/configure-builder/
 will these help?

**Reply by ulysses-you** [reply]
You can open a issue first and push a pull request which is tested on your local. Then we can verify it and merge/push it to dockerhub if it works.

**Reply by phyyou** [reply]
Thanks @yaooqinn, @ulysses-you.

I will soon work on this and create an issue and a PR.

**Comment by yaooqinn** [comment]
SGTM
